modules/mod_translation/mod_translation.erl

```diff
@@ -437,7 +428,7 @@ valid_config_language(Code, Context) ->
 
 
 %% @doc Add a language to the i18n configuration
--spec language_add(atom(), boolean(), #context{}) -> #context{}.
+-spec language_add(atom() | binary(), boolean(), #context{}) -> #context{}.
```

Comment:
The spec in line 431 uses a record, please define a type for the record and use that instead.